draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -346,6 +347,19 @@ full backoff mechanism must be identical to that of TCP
 backoff mechanisms that would give flows with different round-trip
 times comparable capacity during backoff.
 
+### Operation with the Envelope set by Circuit Breakers {{#circuit-breakers}}
+
+A network transport circuit breaker {{!RFC8084}} is an automatic mechanism
+that is used by some equipment in the network to continuously monitor a
+flow or aggregate set of flows.  The mechanism seeks to detect when the
+flow(s) experience persistent excessive congestion, and when detected,
+to terminate (or significantly reduces the rate of) the flow(s).
+
+A well-designed congestion control algorithm ought to react before
+a flow uses excessive resources and therefore
+needs to operate within the envelope set by network transport circuit
+breaker algorithms.
+
```

Comment:
How does an endpoint know that a "circuit breaker" is deployed, and what envelope is being used? How wide is the deployment? Are we requiring that CCA developers test with circuit breakers?

Reply:
RFC8083 is entirely about RTP/RTCP, and the algorithms rely on analyzing the RTCP messages to evaluate packet losses, etc. I think whatever we say here has to be qualified by "if algorithms are designed to control RTP transmissions [cite], they need to remain within the envelope, etc."

Reply:
I do not understand the comment: if RFC8084 is deployed - then the CCs will be subject to the envelope of this circuit breaker. The text does not explicitly state that they ought to be tested though, maybe it could?  NOTE: that RFC 8084 is NOT about RTP. That topic is dealt with RFC8083, (which is described in the section about multimedia).

Reply:
Saying the level of deployment is a judgment call I'd rather not venture into. I'd be happy though personally to include these in the "special cases" section, if you feel this is more appropriate. 
In reality, the envelope ought not to be crossed by any new algorithm that follows the general rules we laid out, but I do think the original comment is valid: I think the IETF has already provided an envelope and somewhere it ought to appear in this spec.

Reply:
I think that moving the mention of circuit breakers to a special case is an OK compromise. I did a quick look at citations, and Google Scholar gives 31 results -- which is not bad, but looks to mostly come from IETF drafts and RFC. For comparison, the FQ-Codel spec has 149 citations on Google scholar. It is also referenced in user manuals of commercial routers (e.g., Aruba), while I did not find that for RFC8084. So, some interest, but not at the same level as popular AQM techniques. It does look like a special case.

Reply:
OK, moved as suggested and framed as a special case.